.changeset/swift-readers-pretend.md

```diff
@@ -0,0 +1,6 @@
+---
+"@yamada-ui/list": patch
+"@yamada-ui/theme": patch
```

Comment:
I think it needs to be major, not patch.

Reply:
I fixed it in this commit.
[dd144d9](https://github.com/yamada-ui/yamada-ui/commit/dd144d935051426f9ba63ad362d042b5f92cfe60)